crates/solidity/testing/snapshots/cst_output/SourceUnit/partial_definition/generated/0.4.11.yml

```diff
@@ -6,12 +6,23 @@ Source: >
 
 Errors: # 1 total
   - >
-    Error: Expected end of input.
+    Error: Expected FallbackKeyword or Identifier or ReceiveKeyword.
```

Comment:
Is this error location correct? It is expecting `FallbackKeyword or Identifier or ReceiveKeyword` at the beginning of the file, instead of at the end.

Reply:
That's because the error range covers the entire production IIUC.

Reply:
The error is produced while parsing the `FunctionDefinition` production, not `SourceUnit`. Am I missing something?

Reply:
I will have to check, but that shows a across-multiple-lines error, so I'll have to investigate. I'll create an issue for this because IMO not worth holding up this PR.

Reply:
#510